src/phoenix/db/migrations/env.py

```diff
@@ -15,7 +15,7 @@
 # Interpret the config file for Python logging.
 # This line sets up loggers basically.
 if config.config_file_name is not None:
-    fileConfig(config.config_file_name)
+    fileConfig(config.config_file_name, disable_existing_loggers=False)
```

Comment:
this was sticking - causing errors to get stuck

Reply:
That is a weird default.